inst/extdata/mass-balance-settings.json

```diff
@@ -0,0 +1,100 @@
+{
+    "MassBalancePlots": [
+        {
+            "Name": "Name of mass balance sub-section (optional)",
+            "Molecules": [
+                "Aciclovir"
+            ],
+            "Groupings": [
+                {
+                    "Name": "Intracellular",
+                    "Include": [
+                        "Organism|**|Intracellular|Aciclovir*"
+                    ],
+                    "Exclude":[
+                        "Organism|Lumen|*|Aciclovir*"
+                    ]
+                },
+                {
+                    "Name": "Plasma & Blood Cells",
+                    "Include": [
+                        "Organism|**|Plasma|Aciclovir*",
+                        "Organism|**|BloodCells|Aciclovir*"
+                    ],
+                    "Exclude":[
+                        "Organism|Lumen|*|Aciclovir*"
+                    ]
+                },
+                {
+                    "Name": "Interstitial",
+                    "Include": [
+                        "Organism|**|Interstitial|Aciclovir*"
+                    ],
+                    "Exclude":[
+                        "Organism|Lumen|*|Aciclovir*"
+                    ]
+                },
+                {
+                    "Name": "Lumen",
+                    "Include": [
+                        "Organism|Lumen|*|Aciclovir*"
+                    ],
+                    "Exclude":[
+                        "Organism|Lumen|Feces|Aciclovir*"
+                    ]
+                },
+                {
+                    "Name": "Excreted",
+                    "Include": [
+                        "Organism|Kidney|Urine|Aciclovir*",
+                        "Organism|Lumen|Feces|Aciclovir*"
+                    ]
+                },
+                {
+                    "Name": "Rest of Aciclovir",
+                    "Include": [
+                        "Organism|**|Aciclovir*"
+                    ]
+                }
+            ]
+        },
+        {
+            "Name": "Aciclovir Mass Balance focus on Fat",
+            "Molecules": [
+                "Aciclovir"
+            ],
+            "Groupings": [
+                {
+                    "Name": "Fat",
+                    "Include": [
+                        "Organism|Fat|*|Aciclovir",
+                        "Organism|Fat|*|Aciclovir"
```

Comment:
why do we have `"Organism|Fat|*|Aciclovir"` 2 times in the list?

Reply:
Copy/paste typo or copilot AI auto fill. I'll remove